packages/cozy-stack-client/src/ContactsCollection.js

```diff
@@ -31,6 +33,28 @@ class ContactsCollection extends DocumentCollection {
     }
     return col
   }
+
+  /**
+   * Destroys a contact
+   *
+   * If the contact is linked to accounts, it will be trashed instead of being
+   * destroyed.
+   *
+   * @param  {object} contact - Contact to destroy. IT MUST BE THE FULL CONTACT OBJECT
```

Comment:
It would still be nice to have a IOCozyContact :wink:

Reply:
Tried something what do you think about it ?

Reply:
Wow, this is very complete, thank you! :heart: 
Now, I'm wondering how we could link the doctype documentation to those types, it would great to have some automatic integration, so we don't need to have 2 places to update when a doctype attribute changes.
Maybe some script `yarn auto-update-types` that would fetch the cozy-doctype repo and autofill the types.js ?

Reply:
Or maybe the source of trush could be the ts types ? And then we could generate an beautiful online documentation ?